src/openforms/fixtures/admin_index_unlisted.json

```diff
@@ -1,5 +1,6 @@
 [
     "config.RichTextColor",
+    "config.MapTileLayer",
```

Comment:
These should go in the configuration group, since the unlisted stuff doesn't get displayed to non-superusers, I think

(and that's probably a problem for the rich text colors)

Reply:
Ahhh okay, yeah that would be a big problem..

Reply:
For now i've left RichTextColors in the unlisted file. I'll add another PR to change that